templates/javascript-ethers/README.md

```diff
@@ -0,0 +1,122 @@
+# Beacon Reader Example
+
+> A starter project for reading beacon values from a smart contract
+
+This project is composed of the following:
+
+1. A test that uses a mocked RrpBeaconServer to simulate reading a beacon value
+   from a smart contract.
+2. A script to deploy the BeaconReaderExample contract to any network. This
+   script will set the address of the RrpBeaconServer contract deployed on the
+   selected network and after the script finishes it will display the address of
+   the contract deployed but also save this address to `deployments` directory
+   on the root of this repo (except for hardhat network since this network is
+   started and stop just for the duration of the script execution).
+3. A script to read the beacon value from the deployed contract. If selected
+   network is not `hardhat` or `localhost` then this address must be then
+   whitelisted by the RrpBeaconServer manager account prior to reading the
+   beacon value.
+
+## Instructions
+
+Run the following to install the dependencies
+
+```sh
+npm install
+# or
+yarn install
+```
+
+Now depending on the network you are using you will need to do the following:
+
+### Default network
+
+The simplest way to test this project is by using the integrated
+[Hardhat network](https://hardhat.org/hardhat-network/), in which case no extra
+steps are needed to run both the test and the scripts.
+
+#### Test
+
+```sh
+npm run test
+# or
+yarn test
+```
+
+#### Script
+
+```sh
+npm run deploy
+npm run read-beacon
+# or
+yarn deploy
```

Comment:
If I run `yarn deploy` I get an error message:
```bash
yarn run v1.22.17
$ hardhat run scripts/deploy.js
Error: Cannot find module '/tmp/tmp.juojYw3Zjg/node_modules/@api3/services/dist/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:320:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:533:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:875:27)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/tmp/tmp.juojYw3Zjg/scripts/deploy.js:9:28)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Yep, because it is not published on npm. When using it as github dependency it is missing the `dist` files with built files.

This will be resolved when @bbenligiray publishes this package.